Dockerfile

```diff
@@ -4,7 +4,7 @@ EXPOSE 80
 
 RUN mkdir /app
 
-# Copying requered files
+# Copy/Copies required files.
```

Comment:
I was a bit unclear, I meant one or the other. 🙂
So
```suggestion
# Copy required files.
```
or
```suggestion
# Copies required files.
```
I think the second alternative is more consistent with the other comments you wrote.